macros/schema_tests/sequential_values.sql

```diff
@@ -0,0 +1,30 @@
+{% macro test_sequential_values(model, column_name, interval=1, datepart=None) %}
```

Comment:
Given all the boiler-room work over in #312, could you add `dispatch` + `default__` here, too?

That takes nothing away from the fact that this is quite elegant SQL, and it runs beautifully on pg/rs/sf/bq :)